spec/system/nsm/disbursements_spec.rb

```diff
@@ -280,6 +378,12 @@
 
       expect(all('table td, table th').map(&:text)).to eq(
         [
+          'Item', 'Total claimed',
```

Comment:
To be the endless repetition of these same snippets across a bunch of tests that aren't about the snippets is just tech debt. It makes the tests harder to read and brittler. Is there any way we can switch out `all('table td, table th').map(&:text)` for something a bit more targeted so these tests can go back to just testing the things they care about?